config/owasp/suppressions.xml

```diff
@@ -1605,4 +1605,84 @@
         <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
         <vulnerabilityName>CVE-2020-1935</vulnerabilityName>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: spring-security-config-5.2.1.RELEASE.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-config@.*$</packageUrl>
+        <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
```

Comment:
can't we make compact/lesser lines for spring security? Seems duplicated lines

Reply:
Addressed removal of duplicate CVEs in this commit: https://github.com/hmcts/rd-profile-sync/pull/239/commits/cd691c20bcc0d939adddabc7800b9c91edc70303

Reply:
Addressed removal of duplicate CPEs in this commit: https://github.com/hmcts/rd-profile-sync/pull/239/commits/21e64b3a8fef3f59d9d346d7efb65f50f1e7dd6b